src/openrct2/Game.h

```diff
@@ -18,6 +18,124 @@
 class Intent;
 struct ParkLoadResult;
 
+// TODO: Move or remove these constants if
+//       we can use the constants defined
+//       in Context.h
+namespace TickConversion
+{
+    constexpr uint32_t kGameUpdateFPS = 40;
+    constexpr float kGameUpdateFPMS =  kGameUpdateFPS / 1000.0f;
+} // namespace TickConversion
+
+template <typename T, typename Tag>
+struct GameTicks
+{
+    using ValueType = T;
+
+    ValueType Value;
+
+    static constexpr GameTicks FromSeconds(uint32_t seconds)
+    {
+        GameTicks<T, Tag> result;
+        result.Value = seconds * TickConversion::kGameUpdateFPS;
+        return result;
```

Comment:
```suggestion
        return FromMilliseconds(seconds * 1000);
```
We don't need to implement this twice.